diff -Nur -x '*.orig' -x '*.rej' e-smith-viewlogfiles-1.8.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/viewlogfiles mezzanine_patched_e-smith-viewlogfiles-1.8.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/viewlogfiles
--- e-smith-viewlogfiles-1.8.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/viewlogfiles	2003-06-25 07:04:06.000000000 +1000
+++ mezzanine_patched_e-smith-viewlogfiles-1.8.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/viewlogfiles	2008-02-10 08:56:58.000000000 +1100
@@ -2,9 +2,10 @@
 -->
 <lexicon lang="en-us">
     <entry>
-        <base>FORM_TITLE</base>
+        <base>View log files</base>
         <trans>View log files</trans>
     </entry>
+
     <entry>
         <base>FIRSTPAGE_DESC</base>
         <trans>
@@ -131,15 +132,4 @@
 	<base>DOWNLOAD_FILE</base>
 	<trans>Preparing to download the logfile {$logfile}.</trans>
     </entry>
-
-    <entry>
-      <base>Administration</base>
-      <trans>Administration</trans>
-    </entry>
-
-    <entry>
-        <base>View log files</base>
-        <trans>View log files</trans>
-    </entry>
-
 </lexicon>
diff -Nur -x '*.orig' -x '*.rej' e-smith-viewlogfiles-1.8.0/root/etc/e-smith/web/functions/viewlogfiles mezzanine_patched_e-smith-viewlogfiles-1.8.0/root/etc/e-smith/web/functions/viewlogfiles
--- e-smith-viewlogfiles-1.8.0/root/etc/e-smith/web/functions/viewlogfiles	2008-02-10 09:00:30.000000000 +1100
+++ mezzanine_patched_e-smith-viewlogfiles-1.8.0/root/etc/e-smith/web/functions/viewlogfiles	2008-02-10 08:57:50.000000000 +1100
@@ -450,7 +450,7 @@
 }
 
 __DATA__
-<form title="FORM_TITLE" header="/etc/e-smith/web/common/head.tmpl" footer="/etc/e-smith/web/common/foot.tmpl">
+<form title="View log files" header="/etc/e-smith/web/common/head.tmpl" footer="/etc/e-smith/web/common/foot.tmpl">
     <page name="Initial"
 	pre-event="print_status_message()">
         <description>FIRSTPAGE_DESC</description>
