diff -Nur -x '*.orig' -x '*.rej' e-smith-viewlogfiles-1.8.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/viewlogfiles mezzanine_patched_e-smith-viewlogfiles-1.8.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/viewlogfiles
--- e-smith-viewlogfiles-1.8.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/viewlogfiles	2008-02-13 23:28:10.000000000 +1100
+++ mezzanine_patched_e-smith-viewlogfiles-1.8.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/viewlogfiles	2008-02-13 23:25:34.000000000 +1100
@@ -111,10 +111,6 @@
 	<trans>Operation</trans>
     </entry>
     <entry>
-	<base>NEXT</base>
-	<trans>Next</trans>
-    </entry>
-    <entry>
 	<base>DOWNLOAD</base>
 	<trans>Download</trans>
     </entry>
