diff -Nur -x '*.orig' -x '*.rej' e-smith-viewlogfiles-1.8.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/viewlogfiles mezzanine_patched_e-smith-viewlogfiles-1.8.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/viewlogfiles
--- e-smith-viewlogfiles-1.8.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/viewlogfiles	2008-04-27 19:03:30.000000000 +0200
+++ mezzanine_patched_e-smith-viewlogfiles-1.8.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/viewlogfiles	2008-04-27 18:59:57.000000000 +0200
@@ -111,10 +111,6 @@
 	<trans>Operation</trans>
     </entry>
     <entry>
-	<base>DOWNLOAD</base>
-	<trans>Download</trans>
-    </entry>
-    <entry>
 	<base>DOWNLOAD_PAGE_DESC</base>
 	<trans><![CDATA[
 	    <p>Your logfile download is now prepared. It will proceed as soon
