diff -Nur -x '*.orig' -x '*.rej' e-smith-viewlogfiles-1.8.0/root/etc/e-smith/web/functions/viewlogfiles mezzanine_patched_e-smith-viewlogfiles-1.8.0/root/etc/e-smith/web/functions/viewlogfiles
--- e-smith-viewlogfiles-1.8.0/root/etc/e-smith/web/functions/viewlogfiles	2008-01-09 10:38:34.000000000 +1100
+++ mezzanine_patched_e-smith-viewlogfiles-1.8.0/root/etc/e-smith/web/functions/viewlogfiles	2008-01-09 10:37:18.000000000 +1100
@@ -242,6 +242,8 @@
         return;
     }
 
+    print "$fullpath: \n";
+
     print $fm->localise("VIEWING_TIME", 
     { time => $fm->gen_locale_date_string() } );
 
