diff -Nur -x '*.orig' -x '*.rej' e-smith-viewlogfiles-1.8.0/root/etc/e-smith/web/functions/viewlogfiles mezzanine_patched_e-smith-viewlogfiles-1.8.0/root/etc/e-smith/web/functions/viewlogfiles
--- e-smith-viewlogfiles-1.8.0/root/etc/e-smith/web/functions/viewlogfiles	2007-10-31 15:28:51.000000000 -0400
+++ mezzanine_patched_e-smith-viewlogfiles-1.8.0/root/etc/e-smith/web/functions/viewlogfiles	2007-10-31 15:28:26.000000000 -0400
@@ -152,12 +152,14 @@
             # or not log files
             foreach (qw(
                 lastlog
+		btmp$
                 wtmp
                 lock
                 (?<!qpsmtpd/)state
                 httpd/ssl_mutex.\d*
                 httpd/ssl_scache.pag
                 httpd/ssl_scache.dir
+		\/config$
             ))
             {
                 return if $path =~ /$_/;
