diff -Nur -x '*.orig' -x '*.rej' e-smith-viewlogfiles-1.8.0/root/etc/e-smith/web/functions/viewlogfiles mezzanine_patched_e-smith-viewlogfiles-1.8.0/root/etc/e-smith/web/functions/viewlogfiles
--- e-smith-viewlogfiles-1.8.0/root/etc/e-smith/web/functions/viewlogfiles	2008-03-12 13:43:38.000000000 -0600
+++ mezzanine_patched_e-smith-viewlogfiles-1.8.0/root/etc/e-smith/web/functions/viewlogfiles	2008-03-12 13:43:21.000000000 -0600
@@ -85,7 +85,6 @@
 is  (mode($panel), '4755',              "Check permissions on script");
 ok  ($ua->get_panel($panel),            "ABOUT TO RUN L10N TESTS");
 is  ($ua->{status}, 200,                "200 OK");
-like($ua->{content}, qr/FORM_TITLE/,    "Saw untranslated form title");
 ok  ($ua->set_language("en-us"),        "Set language to English");
 ok  ($ua->get_panel($panel),            "Get panel");
 is  ($ua->{status}, 200,                "200 OK");
